pytorch/src/main/java/org/bytedeco/pytorch/presets/torch.java

```diff
@@ -2909,13 +100,50 @@ public class torch implements LoadEnabled, InfoMapper {
         Loader.checkVersion("org.bytedeco", "pytorch");
     }
 
+    static void initIncludes(Class thisClass, ClassProperties properties) {
+        // If we are called from Parser, fetch the list of headers to parse from resources
+        if (!Loader.isLoadLibraries()) {
+            for (int i = 3; i < 10; i++) {
+                Class caller = Loader.getCallerClass(i);
+                if (caller == null) return;
+                if (caller == Parser.class) {
+                    properties.put("platform.include", new ArrayList<String>());
+                    Class presets = properties.getEffectiveClasses().get(0);
+                    // We don't want to fill the include list of torch when we are processing torch_cuda
```

Comment:
So, why are you calling it from torch_cuda.class?

Reply:
To read torch_cuda_parselist ? 
Because of the property inheritance I added in the commit, both torch.init() and torch_cuda.init() are called when processing torch_cuda, and we don't want to populate the include property of torch when processing torch_cuda or we will reparse reparse the whole stuff.

Reply:
Please don't do that. Do the inheritance, but modify platform.include differently in the case of torch and torch_cuda.